webodf/lib/runtime.js

```diff
@@ -1834,7 +1834,8 @@ var webodf = {};
             }
             runtime.setCurrentDirectory(path);
             function inner_run() {
-                var script, path, args, argv, result; // hide variables
+                // var script, path, args, argv, result; // hide variables
```

Comment:
Could combine up to line 1841 into this. No real advantage to keeping the declare/assign separate